### ISSUE TITLE
Ability to remove / unset / clear a keyboard shortcut / keybinding

### ISSUE THREAD

**Issue by rootulp** [warpdotdev/Warp#1037]
### Discord username (optional)

rootulp#9326

### Describe the solution you'd like?

```text
I'd like the ability to clear a keyboard shortcut. After clearing a keyboard shortcut, I would expect no keybindings to trigger the command.
```

### Additional context

I use <kbd>CMD + K</kbd> for a keybinding outside Warp. I'd like to completely remove the keybinding associated with the Clear blocks command in order for the keybindings to not conflict.

<img width="502" alt="Screen Shot 2022-04-13 at 1 57 33 PM" src="https://user-images.githubusercontent.com/3699047/163241932-613952ce-bfe1-4891-b64e-12af306b869c.png">

**Comment by rootulp** [comment]
This may manifest as a "Clear" option next to "Reset to default". Another option is to display a menu on right click of the keybinding with the option to "Clear".

**Comment by paustint** [comment]
+1

**Comment by hasanyasin** [comment]
I could see that there is a file (`~/.warp/keybindings.yaml` on MacOS) where custom key bindings are written and I tried setting the shortcut that I want to remove to some invalid value like "null" or empty string, but the application ignored those invalid overrides.

What I ended up as a workaround is to set all the shortcuts to a value that I would never use such as `ctrl-alt-shift-cmd-f16` (which most keyboards can't even do for the lack of F16 key) and the app accepted it as a valid value, hence the shortcut is basically eliminated.

Plus, setting the same value to all the ones you want to disable also works. On the settings tab, it shows those as conflicts but since you want to disable them anyways, you wouldn't care, I believe. :D

**Comment by elviskahoro** [comment]
Gonna close this in favor of #900 so these discussions are on one thread.

Updated the docs with the how the team is currently thinking about all this!
Please check it out!
https://docs.warp.dev/getting-started/privacy

Will share any changes to login and telemetry here.

**Comment by EdmundsEcho** [comment]
I'm not sure if the basis for closing this issue was because Warp now supports "clearing" a keybinding or not.  I'm a new user.  When setting my keybindings, there was no way to literally "clear" whatever entry was associated with a keybinding.  "clear" means `some keybinding -> empty and thus available for use elsewhere`.  Thanks for clarifying.

**Comment by elviskahoro** [comment]
Oh @hasanyasin @EdmundsEcho @rootulp @paustint

**Comment by elviskahoro** [comment]
@hasanyasin @EdmundsEcho @rootulp @paustint Didn't mean to close! Thanks for commenting. I think I accidentally closed both issues (there was a duplicate) instead of just one.

**Comment by elviskahoro** [comment]
Yep. just checked internally and I had left it open. This was definitely an accident!

**Comment by dannyneira** [comment]
Hey Folks, this feature is actively being worked on along with other enhancements to setting KB shortcuts in general, no eta on release but it's on the roadmap 🚀 

Sneak preview ( not final ):

![CleanShot 2023-02-10 at 11 21 04](https://user-images.githubusercontent.com/16809145/218167566-90e22603-b430-4097-a60d-ed8ea451f856.png)
